Sources/PeripheryKit/Indexer/SwiftIndexer.swift

```diff
@@ -44,7 +47,7 @@ public final class SwiftIndexer: Indexer {
                 try indexStore.forEachUnits(includeSystem: false) { unit -> Bool in
                     guard let filePath = try indexStore.mainFilePath(for: unit) else { return true }
 
-                    let file = FilePath.makeAbsolute(filePath)
+                    let file = FilePath.makeAbsolute(filePath, relativeTo: configuration.workingDirectory)
```

Comment:
I'm assuming this is fine since the indexstore will likely have absolute paths when generated from Xcode/the other modes of `periphery` but would like a double-check on that.

Reply:
Yes I think this is ok, I've never come across relative paths working with Xcode and SwiftPM.